.github/workflows/applications.yml

```diff
@@ -59,7 +59,9 @@ jobs:
          cabal-version: ${{ matrix.cabal }}
     - name: Install non-Haskell dependencies (ubuntu)
       if: contains(matrix.os, 'ubuntu')
-      run: sudo apt-get install -y git librocksdb-dev zlib1g-dev libtinfo-dev libsqlite3-dev libz3-dev z3
+      run: |
+          sudo apt-get update
```

Comment:
Does this also need `-y`?

Reply:
I don't think that it has any user interaction. I think, I never put it it there.